collaborators.md

```diff
@@ -1,6 +1,6 @@
 ## Collaborators
 
-level.js is only possible due to the excellent work of the following collaborators:
+`level-js` is only possible due to the excellent work of the following collaborators:
```

Comment:
@vweevers Should we keep this file at all? Merge it with README? Or link to `level/community`?

Reply:
Oh we already are linking from README to `level/community`.

Reply:
We do need a standard solution across the board. The contributors section of `level/community` is slightly outdated, and maybe it'd be better to list contributors per project?

Reply:
Can do that later though.

Reply:
Yes lets do this.